### PR TITLE
Feature/ele2 67 reduce size of post item

### PR DESCRIPTION
**Pull request by skorekm**

### CODE REVIEW

File: scss/components/_search-container.scss [hunk 1]
```diff
@@ -1,3 +0,0 @@
-.list-inset.search-container {
```
Comment (dnaport22) [inline scss/components/_search-container.scss:1]: Whats this?

File: www/index.html [hunk 2]
```diff
@@ -33,6 +34,7 @@
     <script src="js/Plugin/ng-cordova.js"></script>
     <script src="js/Plugin/ngStorage.min.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <script src="js/Plugin/croppie.js"></script>
```
Comment (dnaport22) [inline www/index.html]: Include this on line 36, above ajax.

File: www/js/Controller/Post_item/Postitem_web.js [hunk 3]
```diff
@@ -15,6 +15,21 @@ elephant.controller('PostitemController_web', function($scope,$localStorage ,$io
    */
   var imageToUpload = null;
 
+  var cropContainer = document.getElementById("crop-image-container");
```
Comment (dnaport22) [inline www/js/Controller/Post_item/Postitem_web.js:18]: We have a class which wraps on the docuement.getElementBy__method__(). In order to maintain consistency, I would advice using that. It is located in www/js/Factory/Userinput.js. Example:

www/js/Controller/Form/Login.js
```
var login_data = {
      email: inputVal.getValue(EmailId),
      pass: inputVal.getValue(PassId)
    } 
```

File: www/js/Controller/Post_item/Postitem_web.js [hunk 4]
```diff
@@ -23,17 +38,31 @@ elephant.controller('PostitemController_web', function($scope,$localStorage ,$io
...
-    })
+      //assign url to the cropper
+      cropper.bind({
+          url: imageFile.result,
+          orientation: 1
+      });
+      cropContainer.style.display = "flex";
+    });
+
+    document.getElementById("select-image-button").innerHTML= "Reselect Image";
```
Comment (dnaport22) [inline www/js/Controller/Post_item/Postitem_web.js:54]: Use UserInput class here.

File: www/js/Plugin/croppie.css [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+    width: 100%;
+    height: 100%;
+}
+.croppie-container .cr-image {
+	z-index: -1;
+	position: absolute;
+	top: 0;
+	left: 0;
+	transform-origin: 0 0;
+    max-width: none;
```
Comment (dnaport22) [inline www/js/Plugin/croppie.css:10]: Indentation not consistant

File: www/js/Plugin/croppie.css [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+}
+.croppie-container .cr-boundary {
+	position: relative;
+	overflow: hidden;
+	margin: 0 auto;
+	z-index: 1;
+    width: 100%;
```
Comment (dnaport22) [inline www/js/Plugin/croppie.css:17]: Indentation not consistant.

File: www/js/Plugin/croppie.css [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+}
+.croppie-container .cr-viewport {
+	position: absolute;
+	border: 2px solid #fff;
+	margin: auto;
+	top: 0;
+	bottom: 0;
+	right: 0;
+	left: 0;
+    box-shadow: 0 0 2000px 2000px rgba(0, 0, 0, 0.5);
```
Comment (dnaport22) [inline www/js/Plugin/croppie.css:28]: Indentation not consistant.

File: www/js/Plugin/croppie.css [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+.croppie-container .cr-viewport {
+  -webkit-transform: translateZ(0);
+  -moz-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+}
+
+/*************************************/
+/***** STYLING RANGE INPUT ***********/
+/*************************************/
```
Comment (dnaport22) [inline www/js/Plugin/croppie.css]: Remove third party comments from our code. You have have your own comments if you want to say something.

File: www/js/Plugin/croppie.css [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+/*http://brennaobrien.com/blog/2014/05/style-input-type-range-in-every-browser.html */
+/*************************************/
+
+.cr-slider {
+    -webkit-appearance: none;/*removes default webkit styles*/
+    /*border: 1px solid white; *//*fix for FF unable to apply focus style bug */
```
Comment (dnaport22) [inline www/js/Plugin/croppie.css]: It is not good practice to deploy commented code.

File: www/js/Plugin/croppie.css [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+    width: 16px;
+    border-radius: 50%;
+    background: #ddd;
+    margin-top: -6px;
+}
+.cr-slider:focus {
+    outline: none;
+}
+/*
+.cr-slider:focus::-webkit-slider-runnable-track {
```
Comment (dnaport22) [inline www/js/Plugin/croppie.css]: Remove commented code.

File: www/index.html (hunk 2 above)
Comment (skorekm) [reply www/index.html]: Why is it better to keep it above ajax ?

File: www/js/Controller/Post_item/Postitem_web.js (hunk 3 above)
Comment (skorekm) [reply www/js/Controller/Post_item/Postitem_web.js:18]: Great, done !

File: www/js/Plugin/croppie.css (hunk 5 above)
Comment (skorekm) [reply www/js/Plugin/croppie.css:10]: This is 3rd party code from croppie library, should I correct someone's mistakes ?

File: www/js/Controller/Post_item/Postitem_web.js (hunk 3 above)
Comment (skorekm) [reply www/js/Controller/Post_item/Postitem_web.js:18]: When I change to this method code stops working. We encounter some problem in the library.

**Pull request merged**